crates/sui-graphql-rpc/src/context_data/pg_backend.rs

```diff
@@ -60,12 +60,52 @@ impl GenericQueryBuilder<Pg> for PgQueryBuilder {
     fn get_epoch(epoch_id: i64) -> epochs::BoxedQuery<'static, Pg> {
         epochs::dsl::epochs
             .filter(epochs::dsl::epoch.eq(epoch_id))
+            .select((
```

Comment:
It's kind of annoying that this is required -- @gegaowp, is there a trick to querying for a `Queryable` without enumerating the relevant columns?

Reply:
I think we can if we add the `Selectable` derive on `QueryableEpochInfo`. 
https://github.com/diesel-rs/diesel/pull/2741/files#diff-61a120d6b27e266593a3f04c284b45d06c5298bd18df26a0889b8912a4daab12R100

Reply:
I take that back, it will probably not work because it needs to be associated with a table, and `epochs` table has 19 columns and we only want 18 columns, so might not be possible in general. Maybe @gegaowp has some tricks on how to make this pretty?

Reply:
@wlmyng helped me out and he figured it out how to do it. I will update the PR.